gcpy/file_regrid.py

```diff
@@ -32,74 +24,88 @@ def file_regrid(
         dim_format_in,
         dim_format_out,
         cs_res_out=0,
-        ll_res_out='0x0',
+        ll_res_out="0x0",
         sg_params_in=None,
         sg_params_out=None,
-        vert_params_out=None
+        verbose=False,
+        weightsdir="."
 ):
     """
     Regrids an input file to a new horizontal grid specification
     and saves it as a new file.
 
     Args:
-        fin: str
-            The input filename
-        fout: str
-            The output filename (file will be overwritten if it already exists)
-        dim_format_in: str
-            Format of the input file's dimensions (choose from: classic,
-            checkpoint, diagnostic), where classic denotes lat/lon and
-            checkpoint / diagnostic are cubed-sphere formats
-        dim_format_out: str
-            Format of the output file's dimensions (choose from: classic,
-            checkpoint, diagnostic), where classic denotes lat/lon
-            and checkpoint / diagnostic are cubed-sphere formats
+    -----
+    fin: str
+        The input filename
+    fout: str
+        The output filename (file will be overwritten if it already exists)
+    dim_format_in: str
+        Format of the input file's dimensions (choose from: classic,
+        checkpoint, diagnostic), where classic denotes lat/lon and
+        checkpoint / diagnostic are cubed-sphere formats
```

Comment:
These names are a bit confusing unless you are familiar with GCHP output. Let's leave as is for now. I put comments in a new feature request at https://github.com/geoschem/gcpy/issues/267.

Reply:
Duly noted @lizziel.  I put a TODO comment here in commit 922f233.